packages/contracts-bedrock/src/universal/OptimismMintableERC20.sol

```diff
@@ -19,6 +19,9 @@ contract OptimismMintableERC20 is IOptimismMintableERC20, ILegacyMintableERC20,
     /// @notice Address of the StandardBridge on this network.
     address public immutable BRIDGE;
 
+    /// @notice Decimals of the token
+    uint8 private immutable overridenDecimals;
```

Comment:
The style guide says that immutables should be camel case, could you rename this to `DECIMALS` and have it be private still?